### PR TITLE
Fix D3D11_2 GetResourceTiling return unsupport value

### PR DESCRIPTION
**Pull request by rayfalling**
## Description

See Issue #3200

### CODE REVIEW

File: renderdoc/driver/d3d11/d3d11_device2_wrap.cpp [hunk 1]
```diff
@@ -74,7 +74,8 @@ void WrappedID3D11Device::GetResourceTiling(
   if(m_pDevice2 == NULL)
     return;
 
-  m_pDevice2->GetResourceTiling(pTiledResource, pNumTilesForEntireResource, pPackedMipDesc,
```
Comment (Zorro666) [inline renderdoc/driver/d3d11/d3d11_device2_wrap.cpp:77]: There is a helper function `UnwrapResource` to help i.e.

`m_pDevice2->GetResourceTiling(UnwrapResource(pTiledResource), ...`

**Review by Zorro666 — CHANGES_REQUESTED** [review]
Thank you for this - be good to use the `UnwrapResource` helper.

Locally I added a test to d3d11_parameter_zoo which demonstrates the problem and the test fails before this code change and the test passes after this code .

If it helps you I can land the code fix at the same time as the test?

File: renderdoc/driver/d3d11/d3d11_device2_wrap.cpp (hunk 1 above)
Comment (rayfalling) [reply renderdoc/driver/d3d11/d3d11_device2_wrap.cpp:77]: It's Ok, you can add test first and I will change to use helper funciton

**Review by Zorro666 — APPROVED** [review]
Thank you for the update to use `UnwrapResource`

**Pull request merged**